portfolio/pom.xml

```diff
@@ -41,7 +41,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>phoebek-step-2020</deploy.projectId>
+          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
Seems accidental. Did you mean to do this?

Reply:
I thought that was causing a bug but I changed it back now in the later branches.